src/setup/LockToVotePluginSetup.sol

```diff
@@ -186,7 +188,7 @@ contract LockToVotePluginSetup is PluginSetup {
     {
         // Prepare permissions.
         uint256 helperLength = _payload.currentHelpers.length;
-        if (helperLength != 3) {
+        if (helperLength != 4) {
```

Comment:
@brickpop I would (but it's a personal preference) perform the following actions:

1) Create lock variables to extract the needed contracts from the helper instead of using them directly via `_payload.currentHelpers[INDEX]` in the code. It's easier to track them. At that point you can even remove the comments from the code, given that the name of the variable makes the content explicit.
2) Is there a way to validate that the content of the `_payload.currentHelpers` is indeed what you are expecting it to be? If so, add those sanity checks in addition to the `length` check you already have.